salon_creator/lib/screens/lesson_creation_screen.dart

```diff
@@ -0,0 +1,586 @@
+                      height: screenWidth * 0.563,
+                      width: screenWidth,
+                      fit: BoxFit.cover,
+                      image: FileImage(
+                        File(
+                          thumbnailFile.path,
+                        ),
+                      ),
+                    ),
+                    _buildDeleteButton(_buildDeleteThumbnailConfirmationDialog),
+                  ],
+                ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildMediaFileContainer() {
+    final mq = MediaQuery.of(context).size;
+    final screenWidth = mq.width;
+    return ConstrainedBox(
+      constraints: BoxConstraints(minWidth: screenWidth, minHeight: 50),
+      child: Container(
+        color: Color.fromRGBO(195, 195, 195, 0.3),
+        child: AspectRatio(
+          aspectRatio: 16 / 9,
+          child: _mediaFile == null
+              ? TextButton(
+                  child: Icon(
+                    Icons.add_circle,
+                    size: 64,
+                    color: primaryColor,
+                  ),
+                  onPressed: _showBottomSheet,
+                )
+              : Stack(
+                  alignment: Alignment.topRight,
+                  children: [
+                    Image(
+                      height: screenWidth * 0.563,
+                      width: screenWidth,
+                      fit: BoxFit.cover,
+                      image: FileImage(
+                        File(
+                          _mediaFile.path,
+                        ),
+                      ),
+                    ),
+                    _buildDeleteButton(_buildDeleteMediaFileConfirmationDialog),
+                  ],
+                ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildSalonDetailForms(double screenWidth) {
+    return Container(
+      margin: EdgeInsets.symmetric(
+        horizontal: screenWidth * 0.05,
+      ),
+      child: Column(
+        children: [
+          SizedBox(height: 16),
+          _buildSalonNameForm(),
+          Divider(height: 32),
+          _buildCategoryForm(),
+          Divider(height: 32),
+          _buildContentForm(),
+          Divider(height: 32),
+          _buildResourceFileUploadForm(),
+          Divider(height: 32),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              Text(
+                "公開",
+                style: Theme.of(context).textTheme.headline3,
+              ),
+              Switch(value: _publish, onChanged: _changeSwtich),
+            ],
+          ),
+          SizedBox(height: 16),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildResourceFileUploadForm() {
+    return Column(
+      children: [
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            CustomLabel(title: "リソース"),
+            ElevatedButton(
+              onPressed: () {
+                _filePicker();
```

Comment:
Functions should be named with verbNoun convention. So ideally, this should be `_pickFile()`